docs/gaia/delegator-guide-cli.md

```diff
@@ -0,0 +1,406 @@
+
+Once the `deposit` reaches `minDeposit`, the proposal enters the `voting_period`, which lasts 2 weeks. Any **bonded** Atom hodler can then cast a vote on this proposal. The options are `Yes`, `No`, `NoWithVeto` and `Abstain`. The weight of the vote is based on the amount of bonded Atoms of the sender. If they don't vote, delegator inherit the vote of their validator. However, delegators can override their validator's vote by sending a vote themselves. 
+
+At the end of the voting period, the proposal is accepted if there is more than 50% `Yes` votes (excluding `Abstain ` votes) and less than 33.33% of `NoWithVeto` votes (excluding `Abstain` votes).
+
+### In practice
+
+::: warning
+**Note: These commands need to be run on an online computer. It is more secure to perform them commands using a ledger device. For the offline procedure, click [here](#signing-transactions-from-an-offline-computer).**
+::: 
+
+```bash
+// Submit a Proposal
+// <type>=text/parameter_change/software_upgrade
+// ex value for flag: <gasPrice>=0.0001stake
+
+gaiacli tx gov submit-proposal --title "Test Proposal" --description "My awesome proposal" --type <type> --deposit=10stake --gas auto --gas-prices <gasPrice> --from <delegatorKeyName>
+
+// Increase deposit of a proposal
+// Retrieve proposalID from $gaiacli query gov proposals --status deposit_period
+// ex value for parameter: <deposit>=1stake
+
+gaiacli tx gov deposit <proposalID> <deposit> --gas auto --gas-prices <gasPrice> --from <delegatorKeyName>
+
+// Vote on a proposal
+// Retrieve proposalID from $gaiacli query gov proposals --status voting_period 
+// <option>=yes/no/no_with_veto/abstain
+
+gaiacli tx gov vote <proposalID> <option> --gas auto --gas-prices <gasPrice> --from <delegatorKeyName>
+```
+
+## Signing transactions from an offline computer
+
+If you do not have a ledger device and want to interract with your private key on an offline computer, you can use the following procedure. First, generate an unsigned transaction on an **online computer** with the following command (example with a bonding transaction):
+
+```bash
+// Bond Atoms 
+// ex value for flags: <amountToBound>=10000stake, <bech32AddressOfValidator>=cosmosvaloper18thamkhnj9wz8pa4nhnp9rldprgant57pk2m8s, <gasPrice>=0.001stake
+
+gaiacli tx staking --amount <amountToBond> --validator <bech32AddressOfValidator> --gas auto --gas-prices <gasPrice> --generate-only > unsignedTX.json
+```
+
+Then, copy `unsignedTx.json` and transfer it (e.g. via USB) to the offline computer. If it is not done already, [create an account on the offline computer](#using-a-computer). Now, sign the transaction using the following command:
+
```

Comment:
You should really check the JSON file on the offline computer to ensure you copied what you intended to.

Reply:
You mean manually (`cat unsignedTx.json` ?)

Reply:
Yes, exactly.